pkg/agent/multicast/mcast_controller.go

```diff
@@ -57,17 +58,19 @@ var (
 )
 
 type mcastGroupEvent struct {
-	group net.IP
-	eType eventType
-	time  time.Time
-	iface *interfacestore.InterfaceConfig
+	group  net.IP
+	eType  eventType
+	time   time.Time
+	iface  *interfacestore.InterfaceConfig
+	sender net.IP
 }
 
 type GroupMemberStatus struct {
 	group net.IP
 	// localMembers is a map for the local Pod member and its last update time, key is the Pod's interface name,
 	// and value is its last update time.
 	localMembers   map[string]time.Time
+	remoteMembers  sets.String
```

Comment:
it is better to explain what does remoteMembers mean.

Reply:
Added.